.github/workflows/tacos_unlock.yml

```diff
@@ -90,16 +90,20 @@ jobs:
         uses: ./tacos-gha/.github/actions/setup
         with:
           ssh-private-key: ${{ secrets.ssh-private-key }}
-          # Use author's username when unlocking for closed PRs
+          # Use author's username for the following actions
```

Comment:
# We explicitly list the low-concern actions, during which users will use
# the identity of the original PR author.